tools/test/test_osbuild_image_info.py

```diff
@@ -135,3 +138,37 @@
 def test_read_boot_entries(tmp_path, fake_tree, entries):
     make_fake_tree(tmp_path, fake_tree)
     assert osbuild_image_info.read_boot_entries(tmp_path / "boot") == entries
+
+
+def make_fake_iso(iso_tree, output_dir) -> str:
+    iso_path = os.path.join(output_dir, "image.iso")
+    subprocess.run(["mkisofs", "-o", iso_path, "-R", "-J", iso_tree], check=True)
+    return iso_path
+
+
+def test_analyse_iso_fail_mount(tmp_path):
```

Comment:
(nitpick) strictly speaking this needs the `@pytest.mark.skipif(os.getuid() != 0, reason="root only")` check as well because while it will fail when run as non-root it will fail for the wrong reason (non-root: failed to setup loop device, root: failed to read superblock and we test for the later)

Reply:
Strictly speaking, you are of course right. :sweat_smile: